content/docs/2.0/concepts/authentication.md

```diff
@@ -136,10 +136,10 @@ Every parameter you define in `TriggerAuthentication` definition does not need t
 You can pull information via one or more environment variables by providing the `name` of the variable for a given `containerName`.
 
 ```yaml
-env: # Optional.
-  - parameter: region # Required - Defined by the scale trigger
-    name: my-env-var # Required.
-    containerName: my-container # Optional. Default: scaleTargetRef.containerName of ScaledObject
+env:                              # Optional.
+  - parameter: region             # Required - Defined by the scale trigger
+    name: my-env-var              # Required.
+    containerName: my-container   # Optional. Default: scaleTargetRef.envSourceContainerName of ScaledObject
```

Comment:
```suggestion
    envSourceContainerName: my-container   # Optional. Default: scaleTargetRef.envSourceContainerName of ScaledObject
```

Reply:
TriggerAuthentication has not been changed, the property there is still `containerName`. Imho it makes sense as the property in TA is not confusing as in ScaledObject.

Reply:
Frankly, I didn't check the file - Sorry about that 🙈

Reply:
Not a problem 😄 But if you think that we should change this for TA as well, we can consider that.

Reply:
Nah it's ok!